tools/xabuild/XABuild.cs

```diff
@@ -32,6 +32,8 @@ static int Main ()
 
 		static void CreateConfig (XABuildPaths paths)
 		{
+			if (File.Exists (paths.XABuildConfig))
+				return;
```

Comment:
@dellis1972 the only issue I found yesterday was this `File.Exists` check will break on Windows. The config file always exists, it is `xabuild.exe.config` on Windows and `MSBuild.dll.config` on other platforms.

Maybe we could add a check to `paths.IsWindows` for now? I'm going to see if there is a way to avoid the config file completely today.